e2e/tests/omnibus-sepolia.toml/BFP_Market_Liquidations.e2e.js

```diff
@@ -0,0 +1,665 @@
+const { approveCollateral } = require('../../tasks/approveCollateral');
+const { depositCollateral } = require('../../tasks/depositCollateral');
+const { delegateCollateral } = require('../../tasks/delegateCollateral');
+const { setMintableTokenBalance } = require('../../tasks/setMintableTokenBalance');
+const { getCollateralConfig } = require('../../tasks/getCollateralConfig');
+const { contractRead } = require('../../tasks/contractRead');
+const { contractWrite } = require('../../tasks/contractWrite');
+const { commitBfpOrder } = require('../../tasks/commitBfpOrder');
+const { settleBfpOrder } = require('../../tasks/settleBfpOrder');
+const { getBfpPosition } = require('../../tasks/getBfpPosition');
+const { getBfpDebt } = require('../../tasks/getBfpDebt');
+const { borrowUsd } = require('../../tasks/borrowUsd');
+const { setConfigUint } = require('../../tasks/setConfigUint');
+const { withdrawCollateral } = require('../../tasks/withdrawCollateral');
+
+describe(require('path').basename(__filename, '.e2e.js'), function () {
+  const provider = new ethers.providers.JsonRpcProvider(
+    process.env.RPC_URL || 'http://127.0.0.1:8545'
+  );
+  const accountId = parseInt(`1337${crypto.randomInt(1000)}`);
+  const wallet = ethers.Wallet.createRandom().connect(provider);
+  // const wallet = new ethers.Wallet('0xab', provider);
+  // const accountId = 1337;
+  const address = wallet.address;
+  const privateKey = wallet.privateKey;
+  const marketId = require('../../deployments/extras.json').eth_market_id;
+
+  let snapshot;
+  let oldMinMarginUsd;
+
+  before('Create snapshot', async () => {
+    snapshot = await provider.send('evm_snapshot', []);
+    log('Create snapshot', { snapshot });
+  });
+
+  after('Restore snapshot', async () => {
+    log('Restore snapshot', { snapshot });
+    await provider.send('evm_revert', [snapshot]);
+  });
+
+  it('should sync time of the fork', async () => {
+    await syncTime();
+  });
+
+  it('should create new random wallet', async () => {
+    log({ wallet: wallet.address, pk: wallet.privateKey });
+    assert.ok(wallet.address);
+  });
+
+  it('should set ETH balance to 100', async () => {
+    assert.equal(await getEthBalance({ address }), 0, 'New wallet has 0 ETH balance');
+    await setEthBalance({ address, balance: 100 });
+    assert.equal(await getEthBalance({ address }), 100);
+  });
+
+  it('should create user account', async () => {
+    assert.equal(
+      await getAccountOwner({ accountId }),
+      ethers.constants.AddressZero,
+      'New wallet should not have an account yet'
+    );
+    await createAccount({ wallet, accountId });
+    assert.equal(await getAccountOwner({ accountId }), address);
+  });
+
+  it('should set fWETH balance to 1_000', async () => {
+    assert.equal(
+      await getCollateralBalance({ address, symbol: 'fWETH' }),
+      0,
+      'New wallet has 0 fWETH balance'
+    );
+    const { tokenAddress } = await getCollateralConfig('fWETH');
```

Comment:
unrelated to this PR at all
we do not need fWETH token. We can wrap normal eth. In all other tests in all other chains we do it with normal WETH contract and has special task to get yourself WETH (which calls WETHContract.deposit under the hood)